bring back browser notifications

patosullivan opened the pull request:
If the user has already given us permission to show them notifications and we have some unreads, send the notification.

arthyn commented on ui/src/app.tsx:
```diff
@@ -51,6 +58,23 @@ const AppRoutes = () => {
+        const type = getNotificationType(rope);
+        const ship = bin.topYarn?.con.find(isYarnShip)?.ship;
+        new Notification(`Landscape: ${app}`, {
+          body: `${ship}${bin.topYarn.con[1]}${bin.topYarn.con[2]}`,
```
does this work for all notifications?

patosullivan replied:
I'll play with it and try to break it.

arthyn reviewed (approved):
lgtm, not sure about the body

patosullivan commented:
@arthyn cleaned it up a bit, accounted for the "emph" in the third yarn array item, switched to using the browser's notification preferences, moved the building of the notification out to its own function.

arthyn reviewed (approved):
lgtm
